app/presenters/html_publication_presenter.rb

```diff
@@ -0,0 +1,28 @@
+class HtmlPublicationPresenter < ContentItemPresenter
+  def body
+    content_item["details"]["body"]
+  end
+
+  def contents
+    content_item["details"]["headings"].html_safe
+  end
+
+  def format_sub_type
+    content_item["links"]["parent"][0]["format_sub_type"]
```

Comment:
Maybe a private helper method for `parent` that does the reaching in could simplify this?